devops/ansible/roles/girder/library/test/run.sh

```diff
@@ -1,5 +1,5 @@
 #/bin/bash
 
 ansible-playbook -i .vagrant/provisioners/ansible/inventory/vagrant_ansible_inventory \
-                 --private-key=.vagrant/machines/girder/virtualbox/private_key \
-                 -u vagrant $*
```

Comment:
What's the reasoning behind using tabs in shell scripts?  And, why mix tabs and spaces?  This will make it so this is hard to read unless you're editor has tabstops at 4 spaces, which isn't a common setting.

I'm against tabs as white space except where absolutely required (Makefiles).